CHANGELOG.rst

```diff
@@ -1,5 +1,56 @@
 Changelog
 =========
+1.13 (2020-12-22)
+-----------------
+MLflow 1.13 includes several major features and improvements:
+
+Features:
+
+New fluent APIs for logging in-memory objects as artifacts:
+
+- Add ``mlflow.log_text`` which logs text as an artifact (#3678, @harupy)
+- Add ``mlflow.log_dict`` which logs a dictionary as an artifact (#3685, @harupy)
+- Add ``mlflow.log_figure`` which logs a figure object as an artifact (#3707, @harupy)
+- Add ``mlflow.log_image`` which logs an image object as an artifact (#3728, @harupy)
+
+UI updates / fixes (#3867, @smurching):
+
+- Add model version link in compact experiment table view
+- Add logged/registered model links in experiment runs page view
+- Enhance artifact viewer for MLflow models
+- Model registry UI settings are now persisted across browser sessions
+- Add model version ``description`` field to model version table
+
+Autologging enhancements:
+
+- Introduce utilities for autologging error tolerance / safety (#3682, dbczumar)
+- Add ``exclusive`` configuration option for autologging (#3869, @dbczumar)
+- Set tags on autologged runs for easy identification (and add tags to start_run) (#3847, @dbczumar)
+- Add ``log_models`` configuration for autologging (#3663, @mohamad-arabi)
+
+More features and improvements:
+
+- Retry SQLAlchemy engine creation (#3784, @HCoban)
```

Comment:
On second thought, this sqlalchemy one seems like more of a small fix. Let’s move it.

Reply:
Done!